test_themes/tests/test_new_page_templates.py

```diff
@@ -76,13 +76,15 @@
     re.compile(r'^s_.*'): [
         's_alert_md',
         's_blockquote_with_icon', 's_blockquote',
-        's_carousel_bordered', 's_carousel_default', 's_carousel_rounded',
+        's_carousel_default', 's_carousel_rounded', 's_carousel_boxed',
+        's_carousel_indicators_dots', 's_carousel_indicators_hidden', 's_carousel_controllers_indicators_outside',
+        's_quotes_carousel',
         's_dynamic', 's_dynamic_empty', 's_dynamic_snippet_blog_posts',
         's_blog_posts_effect_marley', 's_blog_post_big_picture',
         's_col_no_bgcolor', 's_col_no_resize',
         's_event_upcoming_snippet',
-        's_image_gallery_cover', 's_image_gallery_indicators_arrows_boxed', 's_image_gallery_indicators_arrows_rounded',
-        's_image_gallery_indicators_dots', 's_image_gallery_indicators_rounded', 's_image_gallery_show_indicators',
+        's_image_gallery', 's_image_gallery_indicators_arrows_boxed', 's_image_gallery_indicators_arrows_rounded',
+        's_image_gallery_indicators_dots', 's_image_gallery_indicators_squared', 's_image_gallery_indicators_rounded', '.s_image_gallery_indicators_hidden', 's_image_gallery_indicators_bars', 's_image_gallery_indicators_outside','s_image_gallery_controllers_outside_arrows_right', 's_image_gallery_controllers_outside',
```

Comment:
`s_image_gallery_indicators_hidden` should not have a `.` in front

Reply:
Good catch ! It's fixed ✅